### DISCUSSION TITLE
Need help with API to get list of dashboards

### DISCUSSION THREAD

**Discussion by AndreMendes960** [apache/superset#27591] [Q&A / Help]
Currently I am using the /api/v1/dashboard/ endpoint to return the full list of dashboards in my app, so the user can select one and then access the embedding. I am trying to use the filter parameters in the query string to filter the returned results from /api/v1/dashboard/.
Currently I have this: https://example.com/api/v1/dashboard/?q=(filters:!((col:id,opr:eq,value:'4'))). With this url, I am able to filter all the dashboards to the only one that has ID 4. However, I want to filter multiple dashboards, for example, the ones that have ID 4,5 and 6. 
Any idea how I can achieve this? I tried
https://example.com/api/v1/dashboard/?q=(filters:!((col:id,opr:eq,value:'4,5'))), but it returns empty because theres no ID = "4,5"
https://example.com/api/v1/dashboard/?q=(filters:!((col:id,opr:in,value:'4,5'))), but I dont think the 'in' operator is supported in the ID filter

**Comment by ameen7626** [comment]
Hi @AndreMendes960 can you explain or show an example code on how you return the full list of dashboards in your app and made the user select one and then access embedding

**Comment by khushbu-sheth** [comment]
I have similar question. I was going through Guide and looks like filter we provide for different columns will be combined. It says 'all conditions must be met' 

https://www.restack.io/docs/superset-knowledge-superset-api-filters-guide

any way to perform OR operation for those filters?
